aiven/internal/service/wait.go

```diff
@@ -160,35 +160,6 @@ func WaitAfterStaticIpsDissassociation(ctx context.Context, d *schema.ResourceDa
 	return nil
 }
 
-func WaitBeforeStaticIpsDissassociation(ctx context.Context, d *schema.ResourceData, m interface{}) error {
```

Comment:
A suggestion: can we rename this to something like: `WaitStaticIPsReadyToBeDissasociated`?